src/assets/scripts/client/speech.js

```diff
@@ -21,12 +21,39 @@ export const speech_init = () => {
     }
 };
 
+export const randomize_pilot_voice = () => {
+    const voices = [
```

Comment:
should be a constant at file level like `SPEECH_VOICES`. same of `SPEECH_HIGHEST_PITCH` and `SPEED_LOWEST_PITCH` (see below).